MPCAutofill/cardpicker/templates/cardpicker/index.html

```diff
@@ -179,16 +175,14 @@ <h5 class="modal-title" id="selectDrivesLabel">Search Settings</h5>
                                                 data-bs-toggle="toggle" id="{{ source.id }}" data-bs-size="sm">
                                     </div>
                                 </td>
-                                <td style="vertical-align: middle"><p>{{ source.id }}</p></td>

```

Comment:
tbh i'm puzzling over the need for this change - this part of the site look identical on my machine between master and your branch?

Reply:
I'm not sure why I changed the if/else part, that must have been really long ago, but I did notice adding the <p> tag around the source.id causes the text to no longer be vertically centered for some reason. I went ahead and rolled back this block except for the <p> tag so that should be good to go!